src/igvfd/schemas/reference_file.json

```diff
@@ -220,6 +220,9 @@
             "submissionExample": {
                 "appscript": "variants",
                 "igvf_utils": "variants"
+            },
+            "assembly": {
```

Comment:
I think this was accidentally added?

Reply:
removed

Reply:
added back because your other comment said this part can be kept

Reply:
Sorry for the confusion, but I think line 224 should not be here (it is within the `content_type` definition which does not make sense to me?). 

And line 248 could be restored depending on whether we want to have a special property description for Reference File. But if we just want the original description then we can keep that deleted.

Reply:
done